build-system/tasks/visual-diff.js

```diff
@@ -428,7 +439,9 @@ async function snapshotWebpages(percy, page, webpages, config) {
         webpage.loading_incomplete_css, webpage.loading_complete_css);
     await percy.snapshot(name, page, SNAPSHOT_OPTIONS);
     await clearExperiments(page);
+    log('travis', colors.green('●'));
```

Comment:
In tests, we print the following kinds of dots:
https://github.com/ampproject/amphtml/blob/42b4e060f85f25db83047d79934d6cc99acf4257/build-system/tasks/karma.conf.js#L92-L104

We've gotten used to assuming that green dots indicate passing tests. I think it can be misleading when you see a series of green dots followed by a red X mark indicating visual diffs.

Maybe print a dot (not green) for each test that was run, and a circle for each test that was skipped? (I assume there's no way to know before hand that a test resulted in visual diffs, or else we could have used the red / green / yellow convention.)

WDYT?

Reply:
I tried a few other options for the dot, do you like any of them?

![image](https://user-images.githubusercontent.com/1839738/42960580-130df1a0-8b41-11e8-810c-dc73ae62becb.png)

Reply:
I wasn't suggesting using a different kind of unicode dot. (it was hard to find one that rendered correctly on all platforms / browsers.)

I was suggesting using a cyan `●` for tests that are run, and a cyan `○` for tests that are skipped. The output will then look like the image below, but will be all cyan.

<img width="518" alt="" src="https://user-images.githubusercontent.com/26553114/42960751-99b29242-8b41-11e8-900e-435dcad3a575.png">

`</bikeshed>`

Reply:
Marking the skipped ones would mean changing the logic of how we filter, and it's not worth the effort. Let's just choose a symbol and move this forward :)
Good options that are likely to be rendered properly everywhere: https://www.fileformat.info/info/unicode/block/geometric_shapes/list.htm

Reply:
Okay, let's just change...

```
log('travis', colors.green('●'));
```
... to...
```
log('travis', colors.cyan('●'));
```

and call it good.